src/models/converters/balances.rs

```diff
@@ -1,9 +1,11 @@
 use crate::models::backend::balances::Balance as BalanceDto;
 use crate::models::service::balances::Balance;
 use crate::providers::info::{TokenInfo, TokenType};
+use crate::config::{native_coin_decimals, native_coin_symbol, native_coin_name};
 
 impl BalanceDto {
     pub fn to_balance(&self, usd_to_fiat: f64) -> Balance {
+        log::error!("AM I even called?");
```

Comment:
please remove this :)

Reply:
oops forgot to get rid of it before pushing :)